src/gt4py/eve/utils.py

```diff
@@ -84,6 +85,15 @@
 T = TypeVar("T")
 
 
+def first(iterable: Iterable[T], *, default: Union[T, NothingType] = NOTHING) -> T:
```

Comment:
we need a test, especially because the only usage in the code will probably disappear